generators/ruby/faraday/config.yaml

```
@@ -0,0 +1,10 @@
+gitRepoId: onfido-ruby
+npmVersion: ${CLIENT_LIBRARY_VERSION}
+enumUnknownDefaultCase: true
+gemName: onfido
+licenseName: MIT
+files:
```

Let's customize user agent also:
```suggestion
httpUserAgent: onfido-ruby/${CLIENT_LIBRARY_VERSION}
files:
```